src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
     children: PropTypes.node,
     className: PropTypes.string,
+    href: PropTypes.string,
     to: PropTypes.string,
-    size: PropTypes.string.isRequired,
   }
 
-  static defaultProps = {
-    appearance: undefined,
```

Comment:
I added it to #2517. I would rather clean it up elsewhere as the classNames are actually pretty easy to track down and I think it will be easier/nicer as a separate patch. There was already some inconsistency in our button usage and I cleaned that up in this patch. The next one can deal with the `<Button>` prop API.